src/main/kotlin/io/emeraldpay/dshackle/config/CompressionConfig.kt

```diff
@@ -0,0 +1,27 @@
+package io.emeraldpay.dshackle.config
+
+class CompressionConfig(
+    var grpc: GRPC = GRPC()
+) {
+    /**
+     * Config example:
+     * ```
+     * compression:
+     *   grpc:
+     *     server:
+     *       enabled: true
+     *     client:
+     *       enabled: false
+     * ```
+     */
+    class GRPC(
+        var serverEnabled: Boolean = false,
```

Comment:
Are we really need configuration?
And in case we need configuration - maybe make it enabled by default?

Reply:
Agree. When do we not want to use compression?

Reply:
Agree. When do we not want to use compression?

Reply:
Good point, yea. So it will be perfectly to update our docs and add this new config.